Mathlib/Data/Set/Intervals/Disjoint.lean

```diff
@@ -253,4 +255,26 @@ theorem iUnion_Iic_eq_Iic_iSup {R : Type*} [CompleteLinearOrder R] {f : ι → R
   @iUnion_Ici_eq_Ici_iInf ι (OrderDual R) _ f has_greatest_elem
 #align Union_Iic_eq_Iic_supr iUnion_Iic_eq_Iic_iSup
 
+theorem iUnion_Iio_of_not_bddAbove_range (hf : ¬ BddAbove (range f)) :
+    ⋃ i, Iio (f i) = univ := by
+  simpa [not_bddAbove_iff, Set.eq_univ_iff_forall] using hf
```

Comment:
`simpa` says that it can be an `iff` lemma.

Reply:
Made it an `iff` lemma. 
Is the name good like this?